new-integration-tests/src/test/resources/exporter/rest_domainqualtrue.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Consider rename the file name to rest_domainqualtrue.yaml to be consistent with yaml file extension with other similar files in resource directory.

Reply:
done